Site/SiteMultipartMailMessage.php

```diff
@@ -337,6 +339,34 @@ public function addBcc($email)
 	}
 
 	// }}}
+	// {{{ public function getCcArray()
+
+	/**
+	 * Gets an array of email addresses for CC
+	 *
+	 * @return arrays $email addresses for CC.
+	 */
+	public function getCcArray()
+	{
+		// array_filter will remove blank values
+		return array_filter($this->cc_list);
```

Comment:
`array_filter` does a simple boolean `false` check against the values in the array by default. I know we normally aren't crazy about checking if `"" == false` because of things like `"0" == false`, but I _think_ in this case it's ok. I'll rewrite to do a better check if you disagree.